apps/condo/domains/organization/utils/testSchema/index.js

```diff
@@ -52,9 +52,9 @@ async function createTestOrganization (client, extraAttrs = {}) {
     const country = DEFAULT_ENGLISH_COUNTRY
     const name = faker.company.companyName()
     const description = faker.company.catchPhrase()
+    const tin = faker.random.alphaNumeric(10)
```

Comment:
It looks like invalid TIN. You need to use a valid TIN generation.

Reply:
I bring some tin generators for those test